Sources/FluentUI_iOS/Components/Command Bar/CommandBarButton.swift

```diff
@@ -87,14 +89,6 @@ class CommandBarButton: UIButton {
             return
         }
 
-        // always update icon and title as we only display one; we may alterenate between them, and the icon may also change
-        let iconImage = item.iconImage
-        let title = item.title
-        let accessibilityDescription = item.accessibilityLabel
-
-        configuration?.image = iconImage
```

Comment:
Is this line still needed to support a button showing different images for the selected and unselected states?

Reply:
I don't believe we support selected/unselected images? the accent image is handled here and selected tint color is handled in `updateStyle`

Reply:
Okay, I wasn't 100% sure. This looks fine if they aren't support.